modules/luci-base/luasrc/dispatcher.lua

```diff
@@ -418,6 +418,13 @@ function dispatch(request)
 		ctx.authsession = sid
 		ctx.authtoken = sdat.token
 		ctx.authuser = sdat.username
+
+		if http.getenv("HTTPS") == "on" then
+			local httpsexpire = conf.main.stricthttps or ""
+			if httpsexpire ~= "" then
+				http.header("Strict-Transport-Security", 'max-age=%s' % { httpsexpire })
```

Comment:
If only interpolating a single element, leave out the extra table:
`http.header("Strict-Transport-Security", 'max-age=%s' % httpsexpire)`